src/main/java/dukeclasses/Task.java

```diff
@@ -0,0 +1,62 @@
+package dukeclasses;
+
+/**
+ * Represents a Task item which needs to be completed. It can be marked as done or unmarked as not done.
+ */
+public class Task {
+
+    private String description;
+    private boolean isDone;
+
+    /**
+     * Class constructor to instantiate an instance of Task.
+     *
+     * @param description String that describes the Task.
+     */
+    public Task(String description) {
+        this.description = description;
+        isDone = false;
+    }
+
+    /**
+     * Sets the current instance of Task as done (if boolean is true) or not done (if boolean is false).
+     *
+     * @param isDone Boolean that indicate whether task is done or not.
+     */
+    public void setDone(boolean isDone) {
+        this.isDone = isDone;
+    }
+
+    /**
+     * Returns the current instance of Task(true if done or false if not done).
+     *
+     * @return Boolean that indicate whether task is done or not.
+     */
+    public boolean getIsDone() {
```

Comment:
The java coding standard mentions 
>  Boolean variables/methods should be named to sound like booleans

Could the method also be named to sound like booleans? Perhaps isDone() ? Even though this is the "same" as the variable, you will not access the variable directly as it is set to private. So the IDE should still be able to help to recommend the isDone() method whenever you try to access it and not the variable.